databases/week2/hw_week2.sql

```diff
@@ -0,0 +1,69 @@
+--Part 1: Working with tasks
+--1. Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+INSERT INTO task (id, title, description, created, updated, due_date, status_id, user_id) VALUES (36, 'Learn HYF Week2 lesson', 'ERD, Normalization, etc.', '2022-08-22 03:41:53', '2022-08-25 09:19:56', '2022-10-11 10:13:42', 2, NULL);
+
+--2. Change the title of a task
+UPDATE task SET title = 'Do HYF week2 Homework' WHERE id = 36;
+
+--3,4. Change a task due date and Change a task status
+UPDATE task SET due_date = '2022-10-03 07:19:56', status_id = 1 WHERE id = 36;
+
+--5. Mark a task as complete
+UPDATE task SET status_id = 3 WHERE id = 36;
+
+--6. Delete a task
+DELETE FROM task WHERE id = 36;
+
+--Part 2: School database
+DROP DATABASE IF EXISTS school;
+CREATE DATABASE IF NOT EXISTS school;
+Use school;
+--Create Class Table
+CREATE TABLE class (
+    id int NOT NULL PRIMARY KEY AUTO_INCREMENT COMMENT 'Primary Key',
+    name VARCHAR(255) NOT NULL COMMENT 'class name',
+    begins DATE NOT NULL COMMENT 'begin Time',
+    ends DATE NOT NULL COMMENT 'end Time'
+) DEFAULT CHARSET UTF8 COMMENT 'classTable';
+
+--Create student TABLE
+CREATE TABLE student (
+    id int NOT NULL PRIMARY KEY AUTO_INCREMENT COMMENT 'Primary Key',
+    name VARCHAR(255) NOT NULL COMMENT 'student name',
+    email VARCHAR(255) NOT NULL COMMENT 'email id',
+    phone VARCHAR(255) NOT NULL COMMENT 'phone Number',
+    class_id INT NOT NULL,
+    CONSTRAINT fk_class FOREIGN KEY(class_id) REFERENCES class (id) ON DELETE CASCADE
```

Comment:
When a given class_id is deleted, are you sure that you want to delete all students that belong to that class_id? That is what happens when you use `ON UPDATE DELETE` :)

Reply:
Yeah, if there is no class with that id the students belong to that class will also be not present